tests/contrib/hooks/test_gcs_hook.py

```diff
@@ -71,10 +71,11 @@ def setUp(self):
             self.gcs_hook = gcs_hook.GoogleCloudStorageHook(
                 google_cloud_storage_conn_id='test')
 
-    def test_storage_client_creation(self):
+    @mock.patch(BASE_STRING.format("GoogleCloudBaseHook._get_credentials"))
```

Comment:
```suggestion
    @mock.patch(BASE_STRING.format("GoogleCloudBaseHook._get_credentials"), return_value = mock.ANY)
```
It seems to me that makes the code easier to read.

Reply:
But I think that it is worth to return some value here.
```suggestion
    @mock.patch(BASE_STRING.format("GoogleCloudBaseHook._get_credentials"), return_value = "CREDENTIALS")
```
`"CREDENTIALS"` is a marker